pkg/apis/internal.admin.acorn.io/v1/quotarequests.go

```diff
@@ -0,0 +1,75 @@
+package v1
+
+import (
+	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	QuotaRequestCondition          = "quota-request"
+	ProjectEnforcedQuotaAnnotation = "acorn.io/enforced-quota"
```

Comment:
Can this be moved to the `labels` package with the other labels and annotations?